src/test/java/org/jabref/logic/importer/fetcher/SpringerFetcherTest.java

```diff
@@ -106,15 +106,21 @@ void searchByQueryFindsEntry() throws Exception {
 
     @Test
     void testSpringerJSONToBibtex() {
-        String jsonString = "{\r\n" + "            \"identifier\":\"doi:10.1007/BF01201962\",\r\n"
-                + "            \"title\":\"Book reviews\",\r\n"
-                + "            \"publicationName\":\"World Journal of Microbiology & Biotechnology\",\r\n"
-                + "            \"issn\":\"1573-0972\",\r\n" + "            \"isbn\":\"\",\r\n"
-                + "            \"doi\":\"10.1007/BF01201962\",\r\n" + "            \"publisher\":\"Springer\",\r\n"
-                + "            \"publicationDate\":\"1992-09-01\",\r\n" + "            \"volume\":\"8\",\r\n"
-                + "            \"number\":\"5\",\r\n" + "            \"startingPage\":\"550\",\r\n"
-                + "            \"url\":\"http://dx.doi.org/10.1007/BF01201962\",\"copyright\":\"©1992 Rapid Communications of Oxford Ltd.\"\r\n"
-                + "        }";
+        String jsonString = """
```

Comment:
This looks odd? Does that still work with \rn

Reply:
This also works because of following `.gitattributes` content:

https://github.com/JabRef/jabref/blob/8b30960d3f9ce8a618cca4eac0366f70f33c3343/.gitattributes#L10